ecommerce/android/Getflix/app/src/main/java/com/example/getflix/models/ProductModel.kt

```diff
@@ -1,24 +1,22 @@
 package com.example.getflix.models
 
-
-import android.os.Parcel
 import android.os.Parcelable
-import com.google.gson.annotations.SerializedName
 import kotlinx.android.parcel.Parcelize
 
 @Parcelize
-data class ProductModel(
-    @SerializedName("id")
-    val id : Int?,
-    @SerializedName("name")
-    val name : String?,
-    @SerializedName("price")
-    val price : String?,
-    @SerializedName("category")
-    val category : String?,
-    @SerializedName("description")
-    val description : String?) : Parcelable {
-
-}
-
+data class ProductModel (
 
+    val id : Int,
+    val name : String,
+    val price : String,
+    val creation_date : String,
+    val image_url : String,
+    val total_rating : Int,
+    val rating_count : Int,
+    val stock_amount : Int,
+    val description : String,
+    val subcategory : String,
+    val category : String,
+    val brand : String,
+    val vendor : String
+) : Parcelable
```

Comment:
I think we can use annotations in order to follow the naming convention of Kotlin. For example, `@SerializedName("stockAmount")`  instead of defining it as stock_amount. It's not a big issue, just want to mention it.

Reply:
Yes, I learned that the json format of product endpoint has changed two days ago, but some endpoints still use the old version of it. So I think we can update it in the coming days, because I'm still not sure.